src/components/book.js

```diff
@@ -1,62 +1,75 @@
-/* eslint-disable */
 import React from 'react';
 import { useDispatch } from 'react-redux';
 import PropTypes from 'prop-types';
-import { booksActions, deleteBook } from '../redux/books/bookSlice';
-import styles from './styles/book.css';
+import { removeBook } from '../redux/books/bookSlice';
 
-const Book = ({
-  id, category, title, author,
-}) => {
+function Book({ title, author, id }) {
   const dispatch = useDispatch();
 
-  const removeBookHandler = (event) => {
-    const { id } = event.target.dataset;
-    dispatch(booksActions.removeBook(id))
-    dispatch(deleteBook(id));
-  }
+  const removeHandler = (id) => {
+    dispatch(removeBook(id));
+  };
+
+  const category = 'Miscellaneous';
+
   return (
-    <div className={styles.item}>
-      <div className={styles.display}>
-      <div className={styles.info}>
-      <p className={styles.school}>{category}</p>
-      <h3 className={styles.bookName}>{title}</h3>
-      <p className={styles.author}>{author}</p>
-      </div>
-      <div className={styles.btns}>
-      <button type='button'>Comments</button>
-      <div className={styles.gery}></div>
-      <button type="button" data-id={id} onClick={removeBookHandler}>Remove</button>
-      <div className={styles.gery}></div>
-      <button type='button'>Edit</button>
-      </div>
-      </div>
+    <li className="display-flex m-3 space-between book-card">
+      <div>
+        <ul>
+          <li>{category}</li>
+          <li><h2>{title}</h2></li>
+          <li>{author}</li>
+        </ul>
+        <ul className="interactions display-flex">
+          <li>
+            <button type="button">
+              Comments
+            </button>
+          </li>
+          <li>
+            <button type="button" onClick={() => { removeHandler(id); }}>
+              Remove
+            </button>
+          </li>
+          <li>
+            <button type="button">
+              Edit
+            </button>
+          </li>
+        </ul>
 
-      <div className={styles.contain}>
-        <div className={styles.ovalContainer}>
-          <div className={styles.Oval} />
+      </div>
+      <div className="display-flex space-between book-card-details">
+        <div className="display-flex">
+          <span className="completed" />
+          {48}
+          %
+          {' '}
+          <br />
+          {' '}
+          Completed
         </div>
         <div>
-          <div className={styles.completion}>98%</div>
-          <div className={styles.completed}>completed</div>
+          <h4>CURRENT CHAPTER</h4>
+          <div>
+            Chapter
+            {}
+            :
+            {}
+          </div>
+          <div>
+            <button className="mt-1" type="button">Update Progress</button>
+          </div>
```

Comment:
- [ ] As stated by the previous reviewer, kindly make sure to style the update progress button and  also give the current chapter text the same color to match with the design guideline

| Your design  | design guideline |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/224478747-ab47ac30-1b35-4e5f-b208-621ce13b141d.png)  | ![image](https://user-images.githubusercontent.com/53402600/224478764-2f7e4678-a858-4ea6-8b7f-5d2e3d5db19e.png) |